Rakefile

```diff
@@ -715,6 +715,10 @@ end
 # Release tasks
 # ------------------------------------------------------------------------------
 
+def github_access_token
+  File.read('.github_access_token').strip rescue nil
```

Comment:
Isn't a similar file from Rainforest used too?

It would be great if we unified this usage (currently we have a file in Rainforest, CocoaPods-app). Should just use the credentials from `.netrc` instead.